warehouse/accounts/models.py

```diff
@@ -83,6 +83,13 @@ class User(SitemapMixin, db.Model):
     two_factor_allowed = Column(Boolean, nullable=False, server_default=sql.false())
     totp_secret = Column(Binary(length=20), nullable=True)
 
+    webauthn = orm.relationship(
+        "Webauthn",
+        backref="user",
+        uselist=False,
```

Comment:
Don't we want this to be a list? Ideally we're supporting any number of u2f tokens right?

Reply:
Yep, that was just copied over and modified from the `TwoFactor` table that we didn't end up using in the TOTP PR, just so I could get something working quickly. I'll be changing it to a one-to-many relationship pretty soon.

Reply:
Wouldn't it be better to create some generic 2Factor relation, so any new 2Factor method can use it?

Reply:
Probably not, since different 2FA methods have different relational properties: TOTPs are 1-1 (one user, one TOTP secret + application), while WebAuthn is 1-N (1 user, N potential WebAuthn credentials). The current WebAuthn relation is 1-1, but that's only because I haven't yet updated it to take advantage of the underlying `webauthn` module's (new) multiple credential support.

Another challenge with using a single relation: we'd have to add additional type/kind checks to the 2FA method being retrieved outside of the invariants that the DB provides implicitly.